embrace-android-sdk/src/main/java/io/embrace/android/embracesdk/comms/api/EmbraceApiService.kt

```diff
@@ -165,20 +154,12 @@ internal class EmbraceApiService(
      * @param networkEvent the event containing the network call information
      */
     override fun sendNetworkCall(networkEvent: NetworkEvent) {
-        logger.logDeveloper(TAG, "sendNetworkCall")
-
-        val url = Unchecked.wrap {
-            EmbraceUrl.getUrl(
-                urlBuilder.getEmbraceUrlWithSuffix("network")
-            )
+        apiUrlBuilders[Endpoint.NETWORK]?.let { url ->
```

Comment:
Is it worth wrapping these types of lookups in checkNotNull? It _should_ never be the case that an endpoint will be null

Reply:
Yeah definitely. Will be doing that later for sure. The sober light of caffeine and y'alls comments makes this obvious lol